plugins/pubIds/doi/locale/es_ES/locale.xml

```diff
@@ -0,0 +1,65 @@
+	<message key="plugins.pubIds.doi.manager.settings.doiAssignJournalWide.confirm">¿Estás seguro de que quieres asignar DOIs a todos los objetos publicados a los que no hayan sido asignados DOIs?</message>
+	<message key="plugins.pubIds.doi.editor.doi">DOI</message>
+	<message key="plugins.pubIds.doi.editor.doiObjectTypeIssue">publicación</message>
+	<message key="plugins.pubIds.doi.editor.doiObjectTypeSubmission">artículo</message>
+	<message key="plugins.pubIds.doi.editor.doiObjectTypeRepresentation">galerada</message>
+	<message key="plugins.pubIds.doi.editor.customSuffixMissing">El DOI no puede ser asignado porque falta el sufijo personalizado.</message>
+	<message key="plugins.pubIds.doi.editor.patternNotResolved">El DOI no puede ser asignado porque contiene un patrón irresoluble.</message>
+	<message key="plugins.pubIds.doi.editor.canBeAssigned">Lo que ve es una previsualización del DOI. Selecciona la casilla de verificación y guarda el formulario para asignar el DOI.</message>
+	<message key="plugins.pubIds.doi.editor.assigned">El DOI es asignado a este {$pubObjectType}.</message>
+	<message key="plugins.pubIds.doi.editor.doiSuffixCustomIdentifierNotUnique">El sufijo DOI proporcionado ya está en uso por otro objeto publicado. Por favor, introduzca un sufijo DOI único para cada objeto.</message>
+	<message key="plugins.pubIds.doi.editor.clearObjectsDoi">Borrar DOI</message>
+	<message key="plugins.pubIds.doi.editor.clearObjectsDoi.confirm">¿Estás seguro de que quieres eliminar el DOI existente?</message>
+	<message key="plugins.pubIds.doi.editor.clearIssueObjectsDoi">Borrar DOIs de Objetos de Publicación</message>
+	<message key="plugins.pubIds.doi.editor.clearIssueObjectsDoi.confirm">¿Estás seguro de que deseas eliminar los DOIs existentes de los objetos de publicación?</message>
+	<message key="plugins.pubIds.doi.editor.clearIssueObjectsDoi.description">Use la siguiente opción para borrar los DOIs de todos los objetos (artículos y galeradas) actualmente planificados para esta publicación.</message>
+	<message key="plugins.pubIds.doi.editor.assignDoi">Asigna el DOI {$pubId} a este {$pubObjectType}</message>
+	<message key="plugins.pubIds.doi.editor.assignDoi.emptySuffix">El DOI no puede ser asignado porque falta el sufijo personalizado.</message>
+	<message key="plugins.pubIds.doi.editor.assignDoi.pattern">El DOI {$pubId} no puede ser asignado porque contiene un patrón irresoluble.</message>
+	<message key="plugins.pubIds.doi.editor.assignDoi.assigned">El DOI {$pubId} ha sido asignado.</message>
+	<message key="plugins.pubIds.doi.manager.settings.doiReassign.description">Si cambia su configuración de DOI, los DOIs que ya han sido asignados no se verán afectados. Una vez esté guardada la configuración del DOI, use este botón para borrar todos los DOIs existentes para que los nuevos ajustes tengan efecto en los objetos existentes.</message>
```

Comment:
If you change your DOI configuration, DOIs that have already been assigned will not be affected. Once the DOI configuration is saved, use this button to clear all existing DOIs so that the new settings will take effect with existing objects.</message>


<message key="plugins.pubIds.doi.manager.settings.doiReassign.description">Si cambia su configuración, los DOIs que ya han sido asignados no se verán afectados. Si lo desea, cuando guarde la configuración del DOI, use este botón para borrar todos los DOIs existentes y que los nuevos ajustes también tengan efecto sobre los identificadores ya asignados.</message>
 +</locale>